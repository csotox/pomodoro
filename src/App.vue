<template>
  <div id="app">
    <router-view class="application__container"/>
  </div>
</template>

<script>
export default {
  beforeMount() {
    this.$store.dispatch('user/login');
  },
};
</script>

<style lang="scss">
@import './styles/reset';
@import './styles/layout';

body {
  margin: 0;
  padding: 0;

  // CSS Variables
  // Colors
  --application-color-primary: #E23434;
  --application-color-accent: #333745;
  --application-color-text: #FFF;
}

* {
  box-sizing: border-box;
}

#app {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
               Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
}
</style>
