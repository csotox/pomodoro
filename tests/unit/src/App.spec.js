import { shallowMount, createLocalVue } from '@vue/test-utils';
import Router from 'vue-router';
import App from '@/App.vue';

const router = new Router();

const localVue = createLocalVue();
localVue.use(Router);

describe('App.vue', () => {
  it('should render properly', () => {
    const app = shallowMount(App, {
      localVue,
      router,
    });
    expect(app).toMatchSnapshot();
  });
});
