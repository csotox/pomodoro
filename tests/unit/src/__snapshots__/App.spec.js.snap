// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should render properly 1`] = `
<div id="app">
  <router-view-stub name="default" class="application__container"></router-view-stub>
</div>
`;
